app/src/test/java/pm/gnosis/heimdall/ui/multisig/MultisigOverviewViewModelTest.kt

```diff
@@ -131,9 +132,9 @@ class MultisigOverviewViewModelTest {
     fun removeMultisigWalletSuccess() {
         val observer = TestObserver.create<Unit>()
         val completable = TestCompletable()
-        given(repository.removeMultisigWallet(eq("Test"))).thenReturn(completable)
+        given(repository.removeMultisigWallet(MockUtils.any())).thenReturn(completable)
```

Comment:
would prefer a `eq(BigInteger.ZERO)` here to make sure that no another wallet is removed